packages/veritone-react-common/src/components/Parts/Buttons/PillButton/styles.scss

```diff
@@ -0,0 +1,37 @@
+.pillButton {
+	border: 1px solid #e0e0e0;
+	border-radius: 20px;
+	padding: 10px;
+	font-size: 14px;
+	display: inline-block;
+
+	&:hover {
+		cursor: pointer;
+	}
+
+	> div {
+		float: left;
+	}
+
+	.gap {
+		width: 1vw;
+		height: 1ex;
+		display: inline-block;
+	}
+
+	.label {
+		white-space: nowrap; 
+		overflow: hidden;
+		text-overflow: ellipsis;
+	}
+	
+	.info {
+		color: #2196f3;
```

Comment:
If you @import 'src/styles/modules/variables'; there is a $blue-1 variable you can use here